docs/source/statistics.mdx

```diff
@@ -54,9 +54,10 @@ curl https://datasets-server.huggingface.co/statistics?dataset=glue&config=cola&
 </curl>
 </inferencesnippet>
 
-The response JSON contains two keys:
-* `num_examples` - number of samples in a split
+The response JSON contains three keys:
+* `num_examples` - number of samples in a split or number of samples in the first N Gb of data (see `partial` field below).
```

Comment:
nit: not more or less correct, as N can be any value, but I think it's easier to relate to current limit of 5 GB

```suggestion
* `num_examples` - number of samples in a split or number of samples in the first N GB of data (see `partial` field below).
```